### PR TITLE
dev-python/numba: version bump to 0.49.1

### PR DESCRIPTION
**Pull request by goduck777**
Package-Manager: Portage-2.3.99, Repoman-2.3.22
Signed-off-by: Chang Liu goduck777@gmail.com

### CODE REVIEW

**Comment by goduck777** [comment]
Package-Manager: Portage-2.3.99, Repoman-2.3.22
Signed-off-by: Chang Liu goduck777@gmail.com

**Comment by epsilon-0** [comment]
@goduck777 I was working on this in https://github.com/gentoo/gentoo/pull/15766  
I'm fine with giving it over to you, but there are a lot of things to change.  
Take a look at the patches and changes I've made and move them over here and let me know.

File: dev-python/numba/numba-0.49.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="NumPy aware dynamic Python compiler using LLVM"
+HOMEPAGE="http://numba.pydata.org/"
```
Comment (charIes17) [inline dev-python/numba/numba-0.49.1.ebuild:11]: Use https

**Comment by goduck777** [comment]
@charIes17 Fixed.

**Comment by goduck777** [comment]
@epsilon-0 Sorry I didn't notice your work. My PR is just version bump, since I didn't use OpenMP or TBB.

File: dev-python/numba/numba-0.49.1.ebuild (hunk 1 above)
Comment (goduck777) [reply dev-python/numba/numba-0.49.1.ebuild]: Fixed.

**Comment by epsilon-0** [comment]
@goduck777 the package is present in ::science overlay

**Comment by goduck777** [comment]
@epsilon-0 I know. Just want to close it.

**Pull request closed without merging**